cinema/migrations/0001_initial.py

```diff
@@ -73,11 +125,33 @@
         migrations.CreateModel(
             name="Ticket",
             fields=[
-                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
                 ("row", models.IntegerField()),
                 ("seat", models.IntegerField()),
-                ("movie_session", models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name="tickets", to="cinema.moviesession")),
-                ("order", models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name="tickets", to="cinema.order")),
+                (
+                    "movie_session",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="tickets",
+                        to="cinema.moviesession",
```

Comment:
Verify that the `moviesession` model exists in the `cinema` app, as it is referenced here as a foreign key. Ensure that the related model is correctly defined and migrated.